src/views/LotteryV2/index.tsx

```diff
@@ -100,10 +94,8 @@ const LotteryV2 = () => {
         </Flex>
       </PageSection>
 
-      <PageSection hasCurvedDivider={false} index={0}>
-        <Flex>
-          <img src="/images/lottery/tombola.png" alt="tombola bunny" height="auto" width="240px" />
-        </Flex>
+      <PageSection hasCurvedDivider={false} index={1} containerProps={{ style: { marginTop: '-32px' } }}>
```

Comment:
I think you can remove the `marginTop: -32px` here - it'll also make the spacing more standardised between the sections.

Reply:
Nah, this is actually needed still cause without it there is a huge gap between this and previous section